### DISCUSSION TITLE
New sign up page seizure problems

### DISCUSSION THREAD

**Discussion by kkmehta03** [community/community#16978] [Other Feature Feedback, Questions, & Ideas]
Hi there! 
In the new github sign up page, there is moving background like start approaching the user. this constant movement causes triggers for focal seizures. when the sign up is complete, there's a faster movement as if a space vehicle has grabbed a good speed. while they're great animations, they cause a lot of discomfort for people with seizures. it caused discomfort for me personally. wanted to know if anyone else is going through something like that?

**Comment by rcandrews-osu** [comment]
I agree to this 100%. I don't even have epilepsy, but that graphic is genuinely painful on a large screen.. It might even be pretty if it was significantly slower & has less contract from black to white.
